services/notifications.go

```diff
@@ -2319,6 +2319,13 @@ func collectMonitoringMachine(
 		}
 	}
 
+	// If more than 90% of users would be notified there is an issue and no one should be notified
```

Comment:
**Minor:** This comment is inaccurate as it is not "more than 90%" but "at least 90%" :monocle_face: 

Furthermore, "there is an issue" would not let me know what this means if I read this piece of code.

Maybe like this:
```suggestion
	// if at least 90% of users would be notified, we expect an issue on our end and no one will be notified
```

Reply:
Sounds good, changed.